lapis/src/main/kotlin/org/genspectrum/lapis/config/ReferenceGenome.kt

```diff
@@ -36,6 +36,11 @@ class ReferenceGenomeSchema(
 
     fun isSingleSegmented(): Boolean = nucleotideSequences.size == 1
 
+    fun getSequenceNameFromCaseInsensitiveName(name: String) =
+        nucleotideSequenceNames[name.lowercase()]?.name
+            ?: geneNames[name.lowercase()]?.name
+            ?: name
```

Comment:
Is `?: name` an expected case or rather an error?

Reply:
The code also pipes metadata field names through here. Maybe this is a little hacky, but I still think that a reasonable default makes more sense than an error here.